tests/platform_helper/providers/test_config_validator.py

```diff
@@ -91,45 +89,21 @@ def test_validate_database_copy_section_failure_cases(
 
     config["extensions"]["our-postgres"]["database_copy"] = database_copy_section
 
-    with pytest.raises(SystemExit):
+    with pytest.raises(ConfigValidatorError) as exc:
```

Comment:
I had to think what `exc` was meant to represent when I read it a few lines down. 

Could we use the more self-explanatory `exception` or `error`?